tests/test_cityTiler.py

```diff
@@ -190,7 +190,20 @@ def test_bridge_split_surface(self):
         objects_type = CityMBridges
         objects_type.set_cursor(cursor)
         city_tiler = CityTiler()
-        city_tiler.args = Namespace(obj=None, loa=None, lod1=False, crs_in='EPSG:3946', crs_out='EPSG:3946', offset=[0, 0, 0], with_texture=False, output_dir=directory, split_surfaces=True)
+        city_tiler.args = Namespace(obj=None, loa=None, lod1=False, crs_in='EPSG:3946', crs_out='EPSG:3946', offset=[0, 0, 0], with_texture=False, output_dir=directory, split_surfaces=True, add_color=False)
+        tileset = city_tiler.from_3dcitydb(cursor, objects_type)
+
+        tileset.write_as_json(directory)
+        cursor.close()
+
+    def test_building_color(self):
+
+        directory = Path("tests/city_tiler_test_data/junk/building_color")
```

Comment:
Still some remaining "junk" in the paths (maybe already fixed in a previous PR?).